pkg/auth/auth.go

```diff
@@ -368,33 +367,33 @@ func (a *Authenticator) CallbackFunc(fn func(loginInfo LoginJSON, successURL str
 		}
 
 		if code == "" {
-			log.Infof("missing auth code in query param")
+			klog.V(4).Info("missing auth code in query param")
 			a.redirectAuthError(w, errorMissingCode)
 			return
 		}
 
 		if urlState != cookieState.Value {
-			log.Errorf("State in url does not match State cookie")
+			klog.Error("State in url does not match State cookie")
 			a.redirectAuthError(w, errorInvalidState)
 			return
 		}
 		ctx := oidc.ClientContext(context.TODO(), a.clientFunc())
 		oauthConfig, lm := a.authFunc()
 		token, err := oauthConfig.Exchange(ctx, code)
 		if err != nil {
-			log.Infof("unable to verify auth code with issuer: %v", err)
+			klog.V(4).Infof("unable to verify auth code with issuer: %v", err)
 			a.redirectAuthError(w, errorInvalidCode)
 			return
 		}
 
 		ls, err := lm.login(w, token)
 		if err != nil {
-			log.Errorf("error constructing login state: %v", err)
+			klog.Errorf("error constructing login state: %v", err)
 			a.redirectAuthError(w, errorInternal)
 			return
 		}
 
-		log.Infof("oauth success, redirecting to: %q", a.successURL)
+		klog.V(4).Infof("oauth success, redirecting to: %q", a.successURL)
```

Comment:
Let's drop the verbosity here, since previously we havent been using `log.Debug()` here.